src/parser/tests/lexer.rs

```diff
@@ -467,10 +467,29 @@ fn lex_block_tag() {
     }
 }
 
+#[test]
+fn lex_filter_tag() {
+    let inputs = vec![
+        "{%- filter tag() %}",
+        "{% filter foo(bar=baz) -%}",
+        "{% filter foo(bar=42) %}",
+        "{% filter foo(bar=baz,qux=quz) %}",
+        "{% filter foo(bar=baz, qux=quz) %}",
+        "{% filter foo ( bar=\"baz\", qux=42 ) %}",
+    ];
+    for i in inputs {
+        assert_lex_rule!(Rule::filter_tag, i);
+    }
+}
+
 #[test]
 fn lex_macro_tag() {
-    let inputs =
-        vec!["{%- macro tag() %}", "{% macro my_block(name) -%}", "{% macro my_block(name=42) %}"];
+    let inputs = vec![
+        "{%- macro tag() %}",
+        "{% macro my_block(name) -%}",
+        "{% macro my_block(name=42) %}",
+        "{% macro foo ( bar=\"baz\", qux=42 ) %}",
```

Comment:
so Ideally space would be allowed inside the parentheses but not between the name and the `(`

Reply:
Unless I'm mistaken, this last test case passes even without this commit (`macro_fn_wrapper` is non-atomic, I suppose that also applies to `macro_fn`):

```
macro_fn        = _{ ident ~ "(" ~ macro_def_args? ~ ")" }
macro_fn_wrapper = !{ macro_fn }
```

Reply:
ah nevermind then!